native_client/java/DeepSpeech-jni/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+# IMPORTANT: Make sure to define your platform with "set(PLATFORM [linux, win32 etc...])" and ensure the enviroment variables JAVA_HOME and CUDA_HOME are set accordingly
+
+cmake_minimum_required(VERSION 3.12.4)
+project(deepspeech-jni)
```

Comment:
As I already gave as feedback on the issue, we don't need the JNI to be that complicated.